CUETools.Codecs.FlaCuda/FlaCudaWriter.cs

```diff
@@ -223,7 +223,7 @@ public object Settings
 				if (value as FlaCudaWriterSettings == null)
 					throw new Exception("Unsupported options " + value);
 				_settings = value as FlaCudaWriterSettings;
-				eparams.flake_set_defaults(_compressionLevel, !_settings.GPUOnly);
+				eparams.Flake_set_defaults(_compressionLevel, !_settings.GPUOnly);

```

Comment:
Please undo. The lower case letter at the beginning of the method name should stay, like in other methods.